MapCore/src/main/java/org/wwarn/mapcore/client/components/customwidgets/facet/FacetSearchableCheckBoxWidget.java

```diff
@@ -57,11 +62,12 @@
     public static final String STYLE_CHECKBOXLIST_ITEM_CHECKED = "searchCheckBoxListItemChecked";
     public static final String STYLE_CHECKBOXLIST_ITEM_DISABLED = "searchCheckBoxListItemDisabled";
     public static final String STYLE_INITIAL_SEARCHBOX_TEXT = "initialSearchBoxText";
-    public static final String STYLE_SELECTION_CLEAR_ANCHOR = "searchableCheckBoxClearAnchor";
```

Comment:
Check the ui binder for this class, as this style class name is still in use just declared in there.
It is preferable not to have css classes in backend code, instead use ui binder, if possible.
For future will be moving to use GSS